### DISCUSSION TITLE
Possible visual refresh

### DISCUSSION THREAD

**Discussion by zachberry** [ucfopen/Obojobo#1664] [Ideas]
Just a mockup of one potential tweak of the design:

![image](https://user-images.githubusercontent.com/73479/107289703-6e386c80-6a33-11eb-9402-f1fc1166fef2.png)

The goals here:

* Simplify and reduce the visual distraction of the nav, allowing users to focus on the content
* Make practice questions more inciting and visually interesting - This is just a quick idea, it's probably _too_ visually interesting and distracting

**Comment by zachberry** [comment]
Same UI but without the gray left-side for the nav:

![image](https://user-images.githubusercontent.com/73479/107292052-2d425700-6a37-11eb-93bf-30210452134f.png)

**Comment by maufcost** [comment]
I like the possible new background for the placeholder of practice questions, and I see what you mean by "distracting". 

We could keep this more engaging background for practice questions and create something more modest for real exam questions, like:
- Only the gradient background (love it!).
- Or just a solid purple color and a slight white-ish border (or vice-versa)
